install/0000_90_machine-api-operator_03_servicemonitor.yaml

```diff
@@ -22,4 +22,43 @@ spec:
   selector:
     matchLabels:
       k8s-app: machine-api-operator
-
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: openshift-machine-api
+  name: machine-api-controllers
+  labels:
+    k8s-app: controller
+spec:
+  namespaceSelector:
+    matchNames:
+      - openshift-machine-api
+  selector:
+    matchLabels:
+      k8s-app: controller
+  endpoints:
+  - port: machine-metrics
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
```

Comment:
How did you decide on the 30s interval for scraping?

Reply:
It varies in range 15s-2m usually in other `ServiceMonitor` instances across my cluster, and I was not sure what is the default value there.